app/components/SecondaryNavBar.js

```diff
@@ -29,14 +29,19 @@ export default class SecondaryNavBar extends React.Component {
     this.props.redirectTo('/users');
   }
 
+  handleManagePermissionsBtnClick()
+  {
+    this.props.redirectTo('/permissions');
```

Comment:
@aksanoble the redirectTo method eventually does that..this is just a wrapper around push in the App.js file..